jigglypuffRL/deeprl/dqn/dqn.py

```diff
@@ -224,6 +283,10 @@ def get_td_loss(self):
         reward = Variable(torch.FloatTensor(reward))
         done = Variable(torch.FloatTensor(done))
 
+        if self.network_type == "cnn":
+            state = state.view(-1, 4, 84, 84)
```

Comment:
This would only work for `BreakoutNoFrameskip-v4` right? Because of the `4`?

Reply:
I don't think so. The history length is standard. Breakout v0 and v4 differ in the fact that v4 is purely deterministic and v0 has 0.25 probability of repeating action. NoFrameSkip or the lack of it will refer to the number of actions after which we're updating. It's normally randomly selected from (2, 5). As such, the code works for Breakout NoFrameSkip v0 and Breakout v0 but I haven't tested v4 waale. Breakout Deterministic takes frameskip to be 4 always so it should run fine.